src/components/plots/rectanglePlot.ts

```diff
@@ -36,12 +36,13 @@ export module Plot {
       var y2Attr = attrToProjector["y2"];
 
       // Generate width based on difference, then adjust for the correct x origin
-      attrToProjector["width"] = (d, i, u, m) => Math.abs(x2Attr(d, i, u, m) - x1Attr(d, i, u, m));
-      attrToProjector["x"] = (d, i, u, m) => Math.min(x1Attr(d, i, u, m), x2Attr(d, i, u, m));
+      attrToProjector["width"] = (d, i, u, m) => Math.abs(x2Attr(d, i, u, m) - x1Attr(d, i, u, m)) || 0;
+      attrToProjector["x"] = (d, i, u, m) => Math.min(x1Attr(d, i, u, m), x2Attr(d, i, u, m)) || 0;
```

Comment:
This has the exact opposite problem of the one in `Pie`: Illegal rectangles will be drawn as size 0, instead of being missing.